src/udiskslinuxfilesystem.c

```diff
@@ -1690,10 +1639,11 @@ handle_unmount (UDisksFilesystem      *filesystem,
 
           g_dbus_method_invocation_return_error (invocation,
                                                  UDISKS_ERROR,
-                                                 get_error_code_for_umount (status, error_message),
+                                                 get_error_code_for_umount (error->code, error->message),
```

Comment:
It isn't the same, `status` was `errno` from the syscall but `get_error_code_for_umount` never used it, so I thought it doesn't matter. I've changed the function to take only the message.